From 814ce75459ce8019eca33ec2fae5bac58b2df076 Mon Sep 17 00:00:00 2001
From: De Huo <De.Huo@windriver.com>
Date: Wed, 24 Jul 2019 15:32:14 +0800
Subject: [PATCH] Add rules accounts_passwords_pam_tally2_deny_root and remove
 accounts_passwords_pam_faillock_deny and
 accounts_passwords_pam_faillock_deny_root from WRLinux1019

Add rule accounts_passwords_pam_tally2_deny_root in WRLinux1019 profile.
Remove rules accounts_passwords_pam_faillock_deny and accounts_passwords_pam_faillock_deny_root
from profile WRLinux1019.
Since WRLinux1019 product use pam_tally2.so instead of pam_faillock.so.

Upstream-Status: Pending

Signed-off-by: De Huo <De.Huo@windriver.com>
---
 wrlinux1019/profiles/draft_stig_wrlinux_disa.profile | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/wrlinux1019/profiles/draft_stig_wrlinux_disa.profile b/wrlinux1019/profiles/draft_stig_wrlinux_disa.profile
index 0efec58..9f461c6 100644
--- a/wrlinux1019/profiles/draft_stig_wrlinux_disa.profile
+++ b/wrlinux1019/profiles/draft_stig_wrlinux_disa.profile
@@ -228,7 +228,6 @@ selections:
     - auditd_data_retention_admin_space_left_action
     - kernel_module_dccp_disabled
     - audit_rules_usergroup_modification
-    - accounts_passwords_pam_faillock_deny
-    - accounts_passwords_pam_faillock_deny_root
     - accounts_passwords_pam_tally2_deny
+    - accounts_passwords_pam_tally2_deny_root
     - package_openssh-server_installed
-- 
1.9.1

